index.html

```diff
@@ -0,0 +1,199 @@
+            </a>
+
+            <div class="col-2 container text-right d-flex"></div>
+            <button class="pelcro-login-button login">LOGIN</button>
+            <button class="pelcro-subscribe-button align-items-start">Subscribe</button>
+            <i class="fas fa-bars"></i>
+          </div>
+          
+        </header>
+      </div>
+    </div>
+
+<footer class="page-footer container-fluid">
```

Comment:
- [ ] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`. 

```suggestion
    <footer class='page-footer container-fluid fixed-bottom'>
```